core/orch_test.go

```diff
@@ -224,15 +224,23 @@ func TestSelectTranscoder(t *testing.T) {
 
 	assert.NotNil(m.liveTranscoders[strm])
 	assert.NotNil(m.liveTranscoders[strm2])
-	assert.Len(m.remoteTranscoders, 9)
+	assert.Len(m.remoteTranscoders, 2)
 
 	// assert transcoder is returned from selectTranscoder and removed from list
```

Comment:
Comment could probably be updated, since the "removed from list" part isn't relevant anymore